packages/mux-player/UPGRADING_V1_to_V2.md

```diff
@@ -0,0 +1,52 @@
+# Upgrading Mux Player from v1.x.x to v2.x.x
```

Comment:
thought(non-blocking): This feels like it has a moderately low probability of being discovered if it's simply an unreferenced md file in the repo. Are there plans of linking to it from docs.mux.com or README.md?

Reply:
Yeah the plan is to use this as a point of reference for linking to from other places.